src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -98,12 +98,12 @@
         :ring?             false}]]
 
      :multiuser
-     [preview-list/view {:type :user :size 20}
+     [preview-list/view {:type :user :size :size/s-20}
       users]
 
      :multinetwork
-     [preview-list/view {:type :network :size 20}
-      (map #(hash-map :profile-picture %) networks)]
+     [preview-list/view {:type :network :size :size/s-20}
```

Comment:
Nice :+1: !!!

But still I don't agree to adding that `s-` preffix because how will we know when it's an `s`, `m` or `l`? I know this is not related to this PR, but wanted to mention it

Reply:
the s- is confusing people so let's remove it. Just :size-20 etc is fine 👍 will be putting this in the docs